fusillade/api/evaluate.py

```diff
@@ -8,8 +8,10 @@
 from fusillade.utils.authorize import assert_authorized, evaluate_policy
 
 
-def evaluate_policy_api(user, body):
-    with AuthorizeThread(user, ['fus:Evaluate'], ['arn:hca:fus:*:*:user']):
+def evaluate_policy_api(token_info, body):
+    with AuthorizeThread(token_info['https://auth.data.humancellatlas.org/email'],
```

Comment:
does the URI for token info need to change based on the ENV?

Reply:
no, this one is fixed. I'll make it config variable in the future